### PR TITLE
add how to install Kerberos  in Windows system

### PR DESCRIPTION
**Pull request by awol2005ex**
add how to install Kerberos  in Windows system

### CODE REVIEW

**Comment by Kimahriman** [comment]
Have you successfully used this? Didn't even know krb5 had Windows libraries, I've only seen references to SSPI.

File: README.md [hunk 1]
```diff
@@ -50,6 +50,13 @@ yum install krb5-libs
 brew install krb5
 ```
 
+#### WIndows
+```bash
+Down and install the Microsoft Kerberos package from https://web.mit.edu/kerberos/dist/
+
+Copy the <INSTALL FOLDER>\MIT\Kerberos\bin\gssapi64.dll to then folder in %PATH% ,and change the name to gssapi_krb5.dll
+```
```
Comment (Kimahriman) [inline README.md:58]:
```suggestion
Download and install the Microsoft Kerberos package from https://web.mit.edu/kerberos/dist/

Copy the `<INSTALL FOLDER>\MIT\Kerberos\bin\gssapi64.dll` file to a folder in %PATH% and change the name to `gssapi_krb5.dll`
```
Do you know if this gets installed into your path automatically? Is it just the file name that is the issue?

**Comment by awol2005ex** [comment]
> Have you successfully used this? Didn't even know krb5 had Windows libraries, I've only seen references to SSPI.

I  had been successfully  to used this in Windows 10

File: README.md (hunk 1 above)
Comment (awol2005ex) [reply README.md:58]: > Do you know if this gets installed into your path automatically? Is it just the file name that is the issue?

It 's  won't  get installed into my path automatically

**Pull request merged**